src/pages/library/datasetBuilder/DatasetBuilder.ts

```diff
@@ -343,7 +386,19 @@ export const ConceptSetSelector = ({
     number: 2,
     onChange,
     objectSets: [
-      { header: 'Concept sets', values: conceptSets },
+      {
+        header: 'Concept sets',
+        values: conceptSets,
+        makeIcon: (value, header) =>
+          h(
+            Clickable,
+            {
+              'aria-label': `Delete ${header}/${value.name}`,
+              onClick: () => updateConceptSets((conceptSets) => _.without([value], conceptSets)),
```

Comment:
Can also remove the lambda here.
```suggestion
              onClick: () => updateConceptSets(_.without([value])),
```